aptos-move/vm-genesis/src/lib.rs

```diff
@@ -360,7 +358,7 @@ fn publish_token_modules(
         let mut bytes = vec![];
         m.serialize(&mut bytes).unwrap();
         session
-            .publish_module(bytes, *module_id.address(), &mut GasStatus::new_unmetered())
+            .publish_module(bytes, *module_id.address(), &mut UnmeteredGasMeter)
```

Comment:
What does this do?

Reply:
It's an dummy gas meter that does not meter anything. All charge operations will succeed.